<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptSettingsDialog</class>
 <widget class="QDialog" name="ScriptSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Script settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="window.qrc">
    <normaloff>:/window/images/alter.png</normaloff>:/window/images/alter.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scriptTab">
      <attribute name="title">
       <string>Frostbite</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Script interpreter</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="scriptInterpreterInput">
                <property name="placeholderText">
                 <string>%PATH%/Ruby</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="scriptInterpreterButton">
                <property name="text">
                 <string>Browse</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Script entry</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="scriptEntryInput">
                <property name="placeholderText">
                 <string>%INSTALL%/scripts/lib/main.rb</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="scriptEntryButton">
                <property name="text">
                 <string>Browse</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Script path</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="scriptPathInput">
                <property name="placeholderText">
                 <string>%INSTALL%/scripts</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="scriptPathButton">
                <property name="text">
                 <string>Browse</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Script extension</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="scriptExtensionInput">
                <property name="placeholderText">
                 <string>.rb</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Script api port</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLineEdit" name="scriptPortInput">
                <property name="placeholderText">
                 <string>auto 1024 - 65535 </string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lichTab">
      <attribute name="title">
       <string>Lich</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Ruby executable</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLineEdit" name="lichRubyInput">
                <property name="placeholderText">
                 <string>%PATH%/Ruby</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="lichRubyButton">
                <property name="text">
                 <string>Browse</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Lich location</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLineEdit" name="lichLocationInput">
                <property name="placeholderText">
                 <string>find: lich.rbw</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="lichLocationButton">
                <property name="text">
                 <string>Browse</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Arguments</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLineEdit" name="lichArgumentsInput">
                <property name="placeholderText">
                 <string>--dragonrealms --frostbite -g $host:$port</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>


     <widget class="QWidget" name="streamingServerTab">
       <attribute name="title">
         <string>Streaming API Server</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QCheckBox" name="streamingServerEnabled">
             <property name="text">
              <string>Enabled</string>
             </property>
            </widget>
           </item>
         <item>
           <widget class="QGroupBox" name="groupBox_9">
             <property name="title">
               <string>Streaming Server TCP port</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                   <item>
                     <widget class="QLineEdit" name="streamingPortInput">
                       <property name="placeholderText">
                         <string>auto 1024 - 65535 </string>
                       </property>
                     </widget>
                   </item>
                 </layout>
               </item>
             </layout>
           </widget>
         </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>         
       </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="window.qrc"/>
 </resources>
 <connections/>
</ui>
